src/System.IO.FileSystem/src/System/IO/EnumerationOptions.cs

```diff
@@ -102,5 +111,10 @@ internal static EnumerationOptions FromSearchOption(SearchOption searchOption)
         /// Set to true to return "." and ".." directory entries.
         /// </summary>
         public bool ReturnSpecialDirectories { get; set; }
+        
+        /// <summary>
+        /// Set to true if the path is normalized, false otherwise.
+        /// </summary>
+        internal bool IsNormalized { get; set; } 
```

Comment:
We're going to have to find another way to plug this in because of the static caching we're relying on for perf. Perhaps internal overloads of the constructors would be a better choice.